korge/src/commonMain/kotlin/com/soywiz/korge/view/animation/ImageAnimationView.kt

```diff
@@ -24,25 +41,28 @@ open class ImageAnimationView(animation: ImageAnimation? = null, direction: Imag
     var direction: ImageAnimation.Direction? = direction
 
     private val computedDirection: ImageAnimation.Direction get() = direction ?: animation?.direction ?: ImageAnimation.Direction.FORWARD
-    private val layers = fastArrayListOf<Image>()
+    private val layers = mutableMapOf<String, Image>()
```

Comment:
Could it be possible to keep the layers list, while still having a `layersByName` using a FastStringMap?
At least for iteration this would be benefical, specially in JS

Reply:
I think that can be done.

Reply:
Done :)